tests/test_frbc_utils.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from python_s2_protocol.common.schemas import CommodityQuantity, NumberRange, PowerRange
+from python_s2_protocol.FRBC.schemas import FRBCOperationMode, FRBCOperationModeElement
+
+from flexmeasures_client.s2.control_types.FRBC.utils import compute_factor
+
+
+@pytest.mark.parametrize("fill_level, fill_rate, factor, fails", [(0.1, 0.3, 1, False)])
+def test_compute_factor(
```

Comment:
Test missing assert statements and a description. What are we actually checking?